CDTS Blazor/Utilities/SessionHelper.cs

```diff
@@ -1,41 +1,76 @@
 ﻿namespace CDNApplication.Utilities
 {
+    using System;
     using Microsoft.AspNetCore.Http;
 
-    [System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1052:Static holder types should be Static or NotInheritable", Justification = "Testing purposes")]
-    [System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "Testing purposes")]
-    public class SessionHelper
+    /// <summary>
+    /// The application's session helper.
+    /// </summary>
+    public sealed class SessionHelper
     {
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1062:Validate arguments of public methods", Justification = "Testing purposes")]
+        // TODO: This should be in a config file
+        private const string DEFAULTLANGUAGE = "en";
```

Comment:
```suggestion
        private const string DefaultLanguage= "en";
```
https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-3.5/ms229043(v=vs.90)?redirectedfrom=MSDN suggests PascalCasing for "Read-only static field"